packages/text-block/src/getPlugins.ts

```diff
@@ -0,0 +1,11 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { defaultPlugins, defaultPluginsWithColumns } from '@frontify/fondue';
+
+export const getPlugins = (columnNumber: number) => {
+    if (columnNumber > 1) {
+        return defaultPluginsWithColumns;
+    } else {
+        return defaultPlugins;
+    }
+};
```

Comment:
```suggestion
export const getPlugins = (columnNumber: number) => columnNumber > 1 ? defaultPluginsWithColumns : defaultPlugins;
```

Reply:
oh true! Removed file and inlined the condition there 👍